weblate/trans/templatetags/translations.py

```diff
@@ -209,12 +210,13 @@ def format_translation(value, language, plural=None, diff=None,
         # Join paragraphs
         content = mark_safe(newline.join(paras))
 
-        parts.append({'title': title, 'content': content})
+        parts.append({'title': title, 'content': content, 'encoded': json.dumps(content)})
```

Comment:
The content here is after highlighting whitespace and similar changes, you should probably use raw_value here.